src/pages/Storefront/Cart.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useShopify } from '../../hooks/useShopify';
+import { CartItem } from 'components/storefront';
+import { Box, HStack, VStack, Heading, Divider, Flex, FormControl, Text, Button, Input } from '@chakra-ui/react';
+// import { useSelector, shallowEqual } from 'react-redux';
+// import { selectors as teamSelectors } from '../../data/reducers/team';
+
+// Cart page that shows the user the items in their cart, allows them to change quantity, and proceed to checkout
+
+const Cart = () => {
+  const { products, checkoutState } = useShopify();
+  // const team = useSelector(teamSelectors.selectTeam, shallowEqual);
+
+  /* previous code that might be userful
+  useEffect(() => {
+    const userID = sessionStorage.getItem('userID');
+    updateCartAttributes(checkoutState.id, [
+      { key: 'userID', value: userID ? userID.toString() : '1' }, // Temporary, so that page doesn't crash
+      // Remove once user info is added to redux
+      { key: 'teamID', value: team.id.toString() },
+      { key: 'teamName', value: team.name },
+    ]);
+  }, [checkoutState.id, team.id, team.name]); // eslint-disable-line react-hooks/exhaustive-deps
+  */
+
+  // Temporary checkout code from last term, will need to udpate
+  const openCheckout = () => {
+    if (checkoutState.webUrl) {
+      console.log('webUrl exists');
+    }
+    // window.open(checkoutState.webUrl) // opens checkout in a new window
+    window.location.replace(checkoutState.webUrl); // opens checkout in same window
+  };
+
+  const applyCoupon = () => {
+    alert('To be implemented');
+  };
+
+  return (
+    <>
+      <Box bg="black" color="black" px="105px">
+        <Heading alignSelf="flex-start" color="white" py={6}>
+          MY SHOPPING BAG
+        </Heading>
+      </Box>
+      <HStack w="100%" h="100%" justifyContent="space-between" alignItems="flex-start" px="105px">
+        <VStack w="100%" alignItems="flex-start" py="64px" pr="45px">
```

Comment:
```suggestion
        <VStack flex={1} alignItems="flex-start" py="64px" pr="45px">
```

is a bit more semantic

Also, I think it would make sense to add `spacing` to this `VStack`, since it looks like we want all the items to be spaced evenly. 

Leverage Chakra's `Stack` component by trying to use spacing on Stack components as much as you can and removing `margins` or `paddings` on individual children for clean styilng